bedrock-devnet/devnet/__init__.py

```diff
@@ -127,12 +127,13 @@ def main():
                      '--build-arg', f'GIT_COMMIT={git_commit}', '--build-arg', f'GIT_DATE={git_date}'],
                     cwd=paths.ops_bedrock_dir, env={
             'PWD': paths.ops_bedrock_dir,
+            'L2_IMAGE': args.l2_image,
             'DOCKER_BUILDKIT': '1', # (should be available by default in later versions, but explicitly enable it anyway)
             'COMPOSE_DOCKER_CLI_BUILD': '1'  # use the docker cache
         })
 
     log.info('Devnet starting')
-    devnet_deploy(paths)
+    devnet_deploy(paths, args)
```

Comment:
```suggestion
    devnet_deploy(paths)
```

I suggest the small changes. You can put this field `l2_image` to paths

Reply:
Thank you,
l2_image is not a path so it should not be in paths